generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -174,10 +168,8 @@ Scenario: Running a 'containingRegex' request that includes anchors ^ and $ shou
        | "b"  |
        | "c"  |
 
-
 Scenario: Running a 'containingRegex' request that includes only anchor ^ should be successful
-     Given there is a field foo
-       And foo is containing regex /^[a-c]{1}/
+     Given foo is containing regex /^[a-c]{1}/
```

Comment:
how does this regex match the scenario description?

Reply:
@mrspaceman would the description for the scenario be more meaningful to be written as:
containingRegex that includes the opening anchor should be successful?

Reply:
when it said "only Anchor" I assumed a regex of `/^/`.
maybe `That does not include the closing anchor $` ?

Reply:
Updated in line with suggestion @mrspaceman